api/server/httputils/write_log_stream.go

```diff
@@ -17,20 +17,13 @@ import (
 
 // WriteLogStream writes an encoded byte stream of log messages from the
 // messages channel, multiplexing them with a stdcopy.Writer if mux is true
-func WriteLogStream(ctx context.Context, w io.Writer, msgs <-chan *backend.LogMessage, config *types.ContainerLogsOptions, mux bool) {
+func WriteLogStream(_ context.Context, w io.Writer, msgs <-chan *backend.LogMessage, config *types.ContainerLogsOptions, mux bool) {
```

Comment:
is there a reason to keep the `context.Context` argument?

Reply:
I'm not sure. It might be fine to remove it.

Reply:
:thinking: might be to satisfy an interface maybe ?

Reply:
I didn't look into it much, we can remove it in a follow up if necessary